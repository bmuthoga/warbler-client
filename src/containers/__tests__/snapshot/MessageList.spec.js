import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import { MessageList } from '../../MessageList'

const props = {
  fetchMessages: jest.fn(),
  messages: [],
  removeMessage: jest.fn(),
  currentUser: {}
}

describe('MessageList Snapshot Tests', () => {
  it('should render the MessageList component', () => {
    const wrapper = shallow(
      <MessageList { ...props } />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
