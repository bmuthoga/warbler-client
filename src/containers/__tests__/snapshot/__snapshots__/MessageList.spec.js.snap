// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageList Snapshot Tests should render the MessageList component 1`] = `
<div
  className="row col-sm-8"
>
  <div
    className="offset-1 col-sm-10"
  >
    <ul
      className="list-group"
      id="messages"
    />
  </div>
</div>
`;
